Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs

```diff
@@ -90,7 +90,7 @@ public MultimirrorDuplicateCommand(TrTransform xf, bool stampMode, BaseCommand p
                 for (int i = 0; i < matrices.Count; i++)
                 {
                     var duplicatedWidget = widget.Clone();
-                    ((Media2dWidget)duplicatedWidget).TwoSided = duplicateAsTwoSided;
+                    if (duplicateAsTwoSided) ((Media2dWidget)duplicatedWidget).TwoSided = true;
```

Comment:
Can you explain to a C# newbie what the difference is here? Is `duplicateAsTwoSided` some type of object that can be null or none or something and thereby work in the `if ()` but can't be used in an assignment because its type has a getter that's failing?

Reply:
In this case casting duplicatedWidget to Media2dWidget results in an "invalid cast" exception if it's not possible. I already check if the cast is valid earlier - that's what the "duplicateAsTwoSided" bool represents. I just need to avoid doing the actual cast if that's not true.

Reply:
(actually - writing this made me realise my PR title is inaccurate - it's not an NRE)

Reply:
thanks. I guess I assumed that the same cast would take place in the `if` as in the assignment, given that the alternative is `true`